src/toil/batchSystems/parasol.py

```diff
@@ -51,7 +51,7 @@ def supportsWorkerCleanup(cls):
         return False
 
     @classmethod
-    def supportsHotDeployment(cls):
+    def supportsRemoteDeployment(cls):
```

Comment:
see comment about deprecating above.,

Reply:
Yes, if you no it is only use by the toil code, then fine.
No, don't change the names.  Rest of code doesn't follow that and that gets complicated.  
Best to just use _ to mean non-public to class or module.

Instead, a guide would be `is it part of the API?